### PR TITLE
feat: Complete coverage for controller

### LINKED ISSUE #496: Complete Coverage for Controller
Extend the existing test for StateController.spec.ts

### PR DESCRIPTION
**Pull request by konstantinabl**
**Description**:

**Related issue(s)**:

Fixes #496

### CODE REVIEW

File: test/unit/controller/StateController.spec.ts [hunk 1]
```diff
@@ -34,23 +34,78 @@ import { CleanUpState } from '../../../src/state/CleanUpState';
...
+      const getStartConfigurationStub = sinon.stub(StateData.prototype, <any>"getStartConfiguration").returns(false);
+      const stateController = new StateController('start');
+      const processExitStub = sinon.stub(process, 'exit');
+
+      // Starting the state machine
+      await stateController.startStateMachine();
+
+      assert.isTrue(processExitStub.calledWith(1));
+
+      //serviceLocatorStub.restore();
```
Comment (natanasow) [inline test/unit/controller/StateController.spec.ts:75]: remove commented line

**Review by natanasow — DISMISSED** [review]
LGTM

**Review by isavov — DISMISSED** [review]

**Comment by stefan-stefanooov** [comment]
Sonarcloud - 2 new issues

**Review by natanasow — APPROVED** [review]

**Review by stefan-stefanooov — APPROVED** [review]

**Review by isavov — APPROVED** [review]
LGTM

**Pull request merged**